subsys/net/lib/nrf_cloud/Kconfig

```diff
@@ -43,6 +43,10 @@ config NRF_CLOUD_SEC_TAG
 	int "Security tag to use for nRF Cloud connection"
 	default 16842753
 
+config NRF_CLOUD_CHECK_CREDENTIALS
+	bool "Enable the ability to check if nRF Cloud credentials are present"
```

Comment:
```suggestion
	bool "Enable checking if nRF Cloud credentials are present"
```

Reply:
enabling this config does not enable checking, it makes the function available to use.  the user would still need to add the function to their application.

Reply:
I stand corrected.